blog/_posts/2022-01-17-scala-3.1.1-released.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: blog-detail
+post-type: blog
+by: Paweł Marks, VirtusLab
+title: Scala 3.1.1 and forward compatibility news
+---
+
+Hello from the Scala 3 team! After a slightly longer than usual pause, we are back with a new release of the Scala compiler and exciting news about improvements in forward compatibility. The delay in the release was caused by urgent bugs that needed backporting and testing, and then the holiday season.
+
+## Improvements is Scala 3.1.1
+
+Scala 3.1.1-RC2 was promoted to a full stable release of the language. Scala 3.1.1 is the first patch version after the 3.1 minor release. It is therefore focused on bug fixes and improvements in overall correctness and stability. Although there are no new additions to the language, you can still spot some noticeable improvements:
```

Comment:
Any mention of Scala 3.1.1-RC2 is supplementary material to omit or move to later, not important information to lead with.

"Scala 3.1.1 is the first patch version after the 3.1 minor release." doesn't add any meaning as far as I can see, I think you could just omit it. Or maybe condense to "Scala 3.1.1 is a patch release focused on bugfixes." I don't think "improvements in overall correctness and stability" adds any meaning and can be omitted.